interface/patient_file/summary/demographics.php

```diff
@@ -1693,64 +1693,53 @@ function setMyPatient() {
   //size check done.
   //extension check done.
   //if any validation needed be added, please add it below.
-  $bool = 0;
+  
+  $image_verified = false;
+  $user_image_absent = true;
+  $extensions = array("jpg", "png", "jpeg"); 
```

Comment:
i see extensions taken here

Reply:
Fewer extensions..., or actually only two.  Can't we detect image by mimetype 'image'?  If not, then we should have a separate function in the system like function_mimetype_class(filename,minsize,maxsize,mimetype). There is  something pretty complex here:  https://github.com/Liroo/react-native-mime-types.
Nothing too wrong with just jpg or png, but it needs a validation warning on extensions allowed, and the file should be evaluated for type.

Reply:
Looks like that is what is being done in sanitize.inc.php sort of.